src/components/AppContent/AppContent.vue

```diff
@@ -23,11 +23,13 @@
 <template>
 	<div id="content" :class="'app-' + appName">
 		<div v-if="!!$slots['navigation']"
-			id="app-navigation">
+			id="app-navigation"
+			:class="navigationClass">
 			<slot name="navigation" />
 		</div>
 		<div v-if="!!$slots['content']"
```

Comment:
Due to that `v-if="!!$slots['content']`, the loading animation is only visible if the content is not empty. That's uncomfortable and I would like to remove this. Is there a need to keep this?

Reply:
@korelstar this is indeed an issue I mentioned with @ChristophWurst . :)

We should detect is slot is not declared, but not if empty indeed.
Because we still want to have app without app-navigation (for the mail account setup for example)

Reply:
I changed the check to `undefined`. Works for me now as intended. :-)